services/spark-operator/metadata.yaml

```diff
@@ -1,10 +1,26 @@
-displayName: Spark
-description: Apache Spark is a fast and general-purpose cluster computing system for big data.
+displayName: Spark Operator
+description: Spark Operator uses Kubernetes custom resources for specifying, running, and surfacing status of Spark applications.
 category:
   - Data Services
 overview: |-
   # Overview
   Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing.
 
-  - [Spark Documentation](https://spark.apache.org/docs/latest/)
+  The Kubernetes Operator for Apache Spark aims to make specifying and running Spark applications as easy and idiomatic as running other workloads on Kubernetes. It uses Kubernetes custom resources for specifying, running, and surfacing status of Spark applications. For a complete reference of the custom resource definitions, please refer to the API Definition. For details on its design, please refer to the design doc. It requires Spark 2.3 and above that supports Kubernetes as a native scheduler backend.
```

Comment:
Adding the overview of Spark **Operator**. contents are straight copied from https://github.com/mesosphere/spark-on-k8s-operator/blob/d2iq-master/README.md